### PR TITLE
Add regional endpoints code samples

### PR DESCRIPTION
**Pull request by jakubrauch**
Tested new code in `dlp/api` with `(cd Snippets.Tests && pwsh runTest.ps1)`.

Please merge when approved.

### CODE REVIEW

File: dlp/api/Snippets/InspectStringRep.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        {
+            Parent = new LocationName(projectId, repLocation).ToString(),
+            Item = new ContentItem
+            {
+                Value = dataValue
+            },
+            InspectConfig = inspectConfig
+        };
+
+        var dlp = new DlpServiceClientBuilder{
```
Comment (amanda-tarafa) [inline dlp/api/Snippets/InspectStringRep.cs:57]:
```suggestion
        var dlp = new DlpServiceClientBuilder
        {
```

File: dlp/api/Snippets.Tests/InspectStringRepTests.cs [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            string repLocation = "us-west1";
+            var infoTypes = new InfoType[] { new InfoType { Name = "PERSON_NAME" } };
+            var customInfoTypes = new CustomInfoType[] { };
+            var response = InspectStringRep.Inspect(Fixture.ProjectId, repLocation, testData, Likelihood.Possible.ToString(), 5, true, infoTypes, customInfoTypes);
+
+            Assert.Contains(response.Result.Findings, f => f.Quote == "Robert");
+        }
+
+        [Fact]
+        public void TestInspectString_NoResults()
```
Comment (amanda-tarafa) [inline dlp/api/Snippets.Tests/InspectStringRepTests.cs:43]: We usually don't test samples so extensively. Sample tests are just for testing that the sample itself works, not for testing Client Library or API functionality, which should be tested elsewhere. These many tests are a maintenance burden and may provoke quota issues, etc.

I think we should be fine with just the first test on this file.

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]
Thanks!

File: dlp/api/Snippets/InspectStringRep.cs (hunk 1 above)
Comment (jakubrauch) [reply dlp/api/Snippets/InspectStringRep.cs]: Updated

File: dlp/api/Snippets.Tests/InspectStringRepTests.cs (hunk 2 above)
Comment (jakubrauch) [reply dlp/api/Snippets.Tests/InspectStringRepTests.cs]: Thanks, removed other four.

**Review by amanda-tarafa — APPROVED** [review]

**Pull request merged**